qualcommax: ipq60xx: WAX610 / WAXY610Y remove unmountable oem partitions on upgrade

ServError opened the pull request:
When using TFTP install method on a fresh unit, oem wifi_fw and ubi_rootfs UBI volumes must be removed or will hang when mounting during boot.

robimarko commented on target/linux/qualcommax/ipq60xx/base-files/lib/upgrade/platform.sh:
```diff
@@ -118,11 +118,16 @@ platform_do_upgrade() {
 		;;
+	netgear,wax610|\
+	netgear,wax610y)
+		CI_UBIPART="rootfs"
```
Isnt this already the default?

ServError replied:
Yes

ServError committed:
qualcommax: ipq60xx: WAX610 remove unmountable oem partitions on upgrade

When using TFTP install method on a fresh unit, wifi_fw and ubi_rootfs UBI volumes must be removed or will hang when mounting.

Signed-off-by: Erik Servili <serverror@serverror.com>
Link: https://github.com/openwrt/openwrt/pull/19215
Signed-off-by: Robert Marko <robimarko@gmail.com>

robimarko commented:
Thanks! Rebased on top of main and merged!
